catalyst_voices/uikit_example/lib/examples/voices_buttons_example.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+enum _ButtonType {
+  filled,
+  outlined,
+  text,
+  textNeutral,
+  textSecondary,
+  icon,
+  iconPrimary,
+  iconFilled,
+  iconTonal,
+  iconOutlined;
+}
+
+enum _ButtonState { normal, disabled }
+
+class VoicesButtonsExample extends StatelessWidget {
+  static const String route = '/buttons-example';
+
+  const VoicesButtonsExample({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Buttons')),
+      body: Padding(
+        padding: const EdgeInsets.all(16),
+        child: ListView(
+          children: _ButtonType.values
+              .map(
+                (type) {
+                  return [
+                    Text(
+                      type.name,
+                      style: Theme.of(context).textTheme.titleMedium,
+                    ),
+                    _ButtonRow(type: type),
+                    _ButtonRow(type: type, addLeadingIcon: true),
+                    _ButtonRow(type: type, addTrailingIcon: true),
+                  ];
+                },
+              )
+              .flattened
+              .expandIndexed(
+                (index, element) => [
+                  if (index != 0) const SizedBox(height: 16),
```

Comment:
how about using ListView.separated instead of ListView, then you don't need to write custom logic for adding spacing and it would more clear